examples/examples/proc_macro.rs

```diff
@@ -28,6 +28,7 @@ use std::net::SocketAddr;
 
 use jsonrpsee::core::{async_trait, client::Subscription, Error};
 use jsonrpsee::proc_macros::rpc;
+use jsonrpsee::types::ReturnTypeSubscription;
```

Comment:
you should use `RpcResult<()>` instead no need to introduce another type it will just annoy users of the API.

I think `RpcResult<()`> is more readable as well.

Reply:
I still don't like the name why not `SubscriptionResult`or something shorter.